bpf/init.sh

```diff
@@ -224,6 +224,10 @@ function bpf_load()
 HOST_DEV1="cilium_host"
 HOST_DEV2="cilium_net"
 
+# Reset all BPF proxy redirections
+rm "/sys/fs/bpf/tc/globals/cilium_proxy4" 2> /dev/null || true
```

Comment:
This seems to be the cause of the test failure. The proxy can't find entries in the map for accepted connections.

Reply:
Which test relies on the proxy map being non-empty after agent restart?